docs/zdgbook/ComponentsAndInterfaces.rst

```diff
@@ -309,27 +308,24 @@ method parameters.
 Checking Implementation
 =======================
 
-You can ask an interface if a certain class or instance that you hand
+You can ask an interface if a certain class that you hand
```

Comment:
Instances can also implement interfaces, not just provide them, though I think this is rare.

Reply:
Actually, I removed "or instance" as the document falsely advised to use `IHello.implementedBy(my_hello_instance)`, which resulted in some trouble, cf.
https://bugs.launchpad.net/zope2/+bug/1197508
and
http://blog.fourdigits.nl/how-to-break-your-plone-site-with-implementedby

Generally, I think it is ok to keep this document on a basic level.

Should we reference to https://zopeinterface.readthedocs.io/en/latest/ ?